components/triplestore/src/test/java/org/trellisldp/triplestore/TriplestoreResourceServiceTest.java

```diff
@@ -237,7 +237,7 @@ public void testRDFConnectionError() throws Exception {
 
         final IRI resource = rdf.createIRI(TRELLIS_DATA_PREFIX + "resource");
         assertThrows(ExecutionException.class, () ->
-                svc.create(resource, mockSession, LDP.RDFSource, rdf.createDataset()).get());
+                svc.create(resource, mockSession, LDP.RDFSource, null, rdf.createDataset()).get());
```

Comment:
You'll find the same issue with tests in this class. Here, feel free to replace `null` with `rdf.createIRI(TRELLIS_DATA_PREFIX)`, which you can put into a private field.